src/ping.ts

```diff
@@ -34,7 +36,13 @@ export class Ping {
         channelId: this.client.config.channelId,
         jid: this.client._stanzaio.jid
       };
-      this.client.logger.warn('Missed a ping.', Object.assign({ error: err }, info));
+
+      /* need to check this because we could have already stopped the pinging. In which case, we don't want to count the missed ping */
+      if (intervalIdAtStart !== this.pingIntervalId) {
+        return this.client.logger.warn('Disregarding a missed ping for an old interval', Object.assign({ error: err }, info));
+      }
```

Comment:
I needed to add this for pings that stanza would reject _after_ we had already called `ping.stop()`. This will drop it on the floor in that case.